documentation/español/protocolo-aodvv2.MD

```diff
@@ -0,0 +1,242 @@
+
+## Manejo de tabla de rutas a partir de mensajes de requerimiento y confirmacion de rutas (RREQ,RREP)
+
+Ahora veremos un ejemplo explicando como se genera el mensaje desde el origen , como se transmite y como es posible obtener tablas de rutas  apartir de estos mensajes.
+
+
+![AODVv2 concept](images/aodv-concept.png "AODVv2 concept")
+
+El nodo A desea enviar datos al nodo E, pero no existe una ruta valida hacia ese nodo, se debe iniciar el proceso de busqueda de ruta.
+
+debemos tener en cuenta lo siguiente:
+
+**1.** 
+- Cada nodo debe mantener 2 contadores.
+  - Un número de secuencia.
+  - Broadcast_id: Se incrementa cuando la fuente genera un nuevo requerimeinto de ruta o RREQ. 
+
+**2.**
+- Tanto el nodo que genera el requerimiento de ruta como el que lo regenera, deben enviar un paquete con la siguiente informacion:
+  - ```<source_addr, source_sequence#, broadcast_id, dest_addr, dest_sequence#, hop_count> ```
+
+- **source_addr**: Es la direccion del generador del requerimiento de ruta u origen del mensaje.
+- **source_sequence#**: Cada nodo mantiene su propio numero de secuencia y se inicia en 1.
+- **broadcast_id**: Cada RREQ tiene su ID unico.
+- **dest_sequence#**: Si no se conoce se deja vacio.
+- **hop_count**: representa la distancia en saltos desde el nodo origen hasta el nodo destino.
+
+**3.**
+- Mensaje de Respuesta para confirmar que se encontro el destino RREP.(unicast), debe tener la siguiente informacion:
+  - ```<source_addr, dest_addr, dest_sequence#, hop_count, lifetime>```
+
+**4.** 
+- Nodos intermedios deberan omitir paquetes duplicados
+
+
+Continuando con el ejemplo, y teniendo en cuenta la informacion suministrada anteriormente, debemos crear unpaquete aodv con la siguiente informacion antes de tratar de transmitir o retransmitir algo.
+
+![alt text](images/aodv-concept1.png "AODV")
+
+**<A, 1, 1, E, , 0>**
+
+- A: Source address.
+- 1: source sequence number.
+- 1: broadcast_id.
+- E: des_addr.
+- no concemos el sequence number del destino al no tener conocimiento de la ruta aun, asi que ese parametro se deja en blanco.
+- 0:hop count representa la distancia hacia el origen del mensaje que en este caso es cero.
+
+al crear dicho paquete se debera transmitir como un mensaje de broadcast para que sea escuchado por todos los vecinos, en este caso en particular el unico vecino adyacente del **nodo A** es el **nodo C**, el cual recibira el paquete creado desde el nodo A.
+
+![alt text](images/aodv-concept2.png "AODV")
+
+el **nodo C** al  no tener informacion sobre el **nodo E** debera incrementar el hop count actualizar su tabla de rutas y retransmitir el requerimiento de ruta RREQ.
+
+el hop count en este caso representa la distancia entre el nodo origen del mensaje (nodo A) y el que recibe el mensaje (nodo C)
+
+```<A, 1, 1, E, , 1>```
+el ultimo parametro del paquete llamado  hop count , nos esta diciendo que desde el nodo C hasta el nodo A hay un salto de distancia. 
```

Comment:
```suggestion
```<A, 1, 1, E, , 1>```

el ultimo parametro del paquete llamado  hop count , nos esta diciendo que desde el nodo C hasta el nodo A hay un salto de distancia. 
```

GitHub gets confused with this

Reply:
it was fixed